### ISSUE TITLE
Unknown type name "DefaultActiveSpeakerPolicy"

### ISSUE THREAD

**Issue by rlebosse** [aws/amazon-chime-sdk-ios#117]
**Describe the bug**

In my **file.m**, I have this at the beginning of the file :

```
#import "NativeMobileSDKBridge.h"
#import <AVFoundation/AVFoundation.h>
#import "RNVideoViewManager.h"
#import <AmazonChimeSDKMedia/AmazonChimeSDKMedia.h>
#import "MeetingObservers.h"
#import <React/RCTUIManager.h>
#import <AmazonChimeSDK/AmazonChimeSDK-Swift.h>

@implementation NativeMobileSDKBridge

static DefaultMeetingSession *meetingSession;
static ConsoleLogger *logger;
static DefaultActiveSpeakerPolicy *activeSpeakerPolicy;
```

While everything is OK with **DefaultMeetingSession** and **ConsoleLogger**,
Xcode shows an error telling : **Unknown type name "DefaultActiveSpeakerPolicy"**

Is there an import that I am missing out, or something else ?

Thanks in advance !

**Test environment Info (please complete the following information):**
 - OS : macOS Catalina 10.15.6
- Xcode 11.7
 - Version AmazonChimeSDK: 0.8.1
 - Version AmazonChimeSDKMedia: 0.6.0

**Comment by tianyux-amz** [comment]
Hi @rlebosse ,

Thank you for reaching out and reporting this issue.

I was able to reproduce your issue. Seems like some classes were not properly exposed in ObjC. We will review all the public Swift classes and make sure they were exposed as expected.

Best,
Tianyu

**Comment by rlebosse** [comment]
I'm just writing a comment to prevent auto close of the issue (since the issue is still here).

**Comment by zhinang-amazon** [comment]
This will be resolved in the next release. Will update here once the release is out.
Fixed by #121

**Comment by zhinang-amazon** [comment]
Hi @rlebosse 
This change is in the [latest SDK release](https://github.com/aws/amazon-chime-sdk-ios/tree/v0.11.1)
Feel free to reopen if this is still an issue for you
